client/packages/invoices/src/Returns/DetailView/Toolbar.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { FC } from 'react';
+import {
+  AppBarContentPortal,
+  Box,
+  InputWithLabelRow,
+  BasicTextInput,
+  Grid,
+  DropdownMenu,
+  useTranslation,
+  DropdownMenuItem,
+  DeleteIcon,
+} from '@openmsupply-client/common';
+import { useReturns } from '../api';
+
+export const Toolbar: FC = () => {
+  const t = useTranslation('distribution');
+  const onDelete = useReturns.document.deleteOutboundRows();
+  //   const { onAllocate } = useOutbound.line.allocateSelected();
+  const { data } = useReturns.document.outboundReturn();
+  const { otherPartyName } = data ?? {};
+  // const { isGrouped, toggleIsGrouped } = useIsGrouped('outboundShipment');
+  //   const [theirReferenceBuffer, setTheirReferenceBuffer] =
+  //     useBufferState(theirReference);
+  //   const { mutateAsync: updateName } = useOutbound.document.updateName();
+
+  //   const isDisabled = useOutbound.utils.isDisabled();
+
+  return (
+    <AppBarContentPortal sx={{ display: 'flex', flex: 1, marginBottom: 1 }}>
+      <Grid
+        container
+        flexDirection="row"
+        display="flex"
+        flex={1}
+        alignItems="flex-end"
+      >
+        <Grid item display="flex" flex={1}>
+          <Box display="flex" flex={1} flexDirection="column" gap={1}>
+            {otherPartyName && (
+              <InputWithLabelRow
+                label={t('label.customer-name')}
```

Comment:
I think this should be supplier name... outbound return, going back to the supplier?